### PR TITLE
[CHANGE] Deduplicate L1A1s and remove from the Opfor arsenal

### PR DESCRIPTION
**Pull request by dijksterhuis**
I've done my best to standardise the values based on the original WEST settings in the arsenal config when deduplicating.

Also, no more Opfor L1A1s (not that anyone abused it).

### CODE REVIEW

File: mission/config/arsenal.hpp [hunk 1]
```diff
@@ -194,21 +183,21 @@ class vn_whitelisted_arsenal_loadouts
...
-			{"vn_l1a1_02_mrk",{4,-1,4,4}},
-			{"vn_l1a1_02_camo",{4,-1,4,4}},
-			{"vn_l1a1_02_bayo",{4,-1,4,4}},
-			{"vn_l1a1_02_gl",{4,-1,4,4}},
-			{"vn_l1a1_03",{4,-1,4,4}},
-			{"vn_l1a1_03_camo",{4,-1,4,4}},
-			{"vn_l1a1_xm148",{4,-1,4,4}},
-			{"vn_l1a1_xm148_camo",{4,-1,4,4}},
-			{"vn_l2a1_01",{4,-1,4,4}},
+			{"vn_l1a1_01",{-1,1,1,-1}},
```
Comment (dijksterhuis) [inline mission/config/arsenal.hpp:186]: This was original disabled for west, but is the most basic L1A1 weapon, so have enabled at the same level as the `vn_l1a1_01_camo`

File: mission/config/arsenal.hpp [hunk 2]
```diff
@@ -194,21 +183,21 @@ class vn_whitelisted_arsenal_loadouts
...
+			{"vn_l1a1_01_camo",{-1,1,1,-1}},
+			{"vn_l1a1_01_bayo",{-1,1,1,-1}},
+			{"vn_l1a1_01_gl",{-1,2,2,-1}},
+			{"vn_l1a1_01_mrk",{-1,-1,-1,-1}}, // unknown so disabled by @dijksterhuis
```
Comment (dijksterhuis) [inline mission/config/arsenal.hpp:190]: I'm guessing this is some sort of marksman reference, but changing these values doesn't seem to do anything.

File: mission/config/arsenal.hpp [hunk 3]
```diff
@@ -194,21 +183,21 @@ class vn_whitelisted_arsenal_loadouts
...
+			{"vn_l1a1_02",{-1,1,1,-1}},
+			{"vn_l1a1_02_camo",{-1,2,2,-1}},
+			{"vn_l1a1_02_bayo",{-1,1,1,-1}},
+			{"vn_l1a1_02_gl",{-1,3,3,-1}},
+			{"vn_l1a1_02_mrk",{-1,-1,-1,-1}}, // unknown so disabled by @dijksterhuis
```
Comment (dijksterhuis) [inline mission/config/arsenal.hpp:195]: I'm guessing this is some sort of marksman reference, but changing these values doesn't seem to do anything.

**Review by dijksterhuis — COMMENTED** [review]
I've set all the independent values to equal west, and all the civilian/east values to be disabled.

**Review by wescottj98 — CHANGES_REQUESTED** [review]
Dont remove any lines from here. if you want to change a given side's access to an item change the numbers accordingly. Refer to the top of the file.

**Comment by dijksterhuis** [comment]
@wescottj98 

> Dont remove any lines from here. if you want to change a given side's access to an item change the numbers accordingly. Refer to the top of the file.

I did refer to the top of the file.

The lines I removed were duplicate entries for the same assets.

Cross reference modified diff lines with the removed diff lines and you should see that I've just deduplicated the entries.

Unless there's some specific reason why they were listed twice? (no-one added a comment about it if so)

**Review by wescottj98 — APPROVED** [review]

**Pull request merged**